.github/workflows/build.yml

```
@@ -10,6 +10,7 @@ name: Build
 
 on:
   pull_request:
+    types: [opened, reopened, synchronize]
```

🤔 I like how these are succinct and implicit. Since per https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#pull_request these are the defaults, I don't see any need to explicitly declare them... 

I also tried removing the `:` to make line 12 `pull_request` and got a formatting complaint from Prettier:

```plaintext
✖ prettier --ignore-unknown --write:
[error] .github/workflows/build.yml: SyntaxError: Implicit map keys need to be on a single line (12:3)
[error]   10 |
[error]   11 | on:
[error] > 12 |   pull_request
[error]      |   ^^^^^^^^^^^^
[error] > 13 |
[error]      | ^
[error] > 14 |   push:
[error]      | ^^^^^^^
[error]   15 |     branches:
[error]   16 |       - main
[error]   17 |
```

[TIL you can use `~` or `null`](https://stackoverflow.com/questions/34089496/empty-field-in-yaml). It seems to work in #309's builds. Let's go with `~` for all of these files instead of the explicit `types` array?

```suggestion
  pull_request: ~
```

---

Had to add this explicitly cause the linter was complaining about empty value, the `~` trick does the job  :1st_place_medal: